crates/sui-types/src/execution_status.rs

```diff
@@ -88,7 +88,9 @@ pub enum ExecutionFailureStatus {
     MoveAbort(MoveLocation, u64),
     #[error(
         "Move Bytecode Verification Error. \
-        Please run the Bytecode Verifier for more information."
+        Please run the Bytecode Verifier for more information. \
+        If applicable, verify that all entry functions' type arguments have been \
```

Comment:
We shouldn't change the wording of this error message -- it can happen in a lot of cases, mostly not to do with type arguments, so we should seek to explain about that particular case in other ways (like you have).